data/korath.txt

```diff
@@ -202,20 +202,71 @@ fleet "Korath Raid"
 		"Korath Chaser" 2
 
 fleet "Korath Home"
-	government "Korath"
-	names "korath"
-	fighters "korath"
-	cargo 3
-	personality
-		timid opportunistic
-	variant 12
-		"Korath World-Ship"
-	variant 12
-		"Korath World-Ship B"
-	variant 12
-		"Korath World-Ship C"
+    government "Korath"
```

Comment:
Your editor ate the tabs that were here, and put spaces instead. Same applies for some other files.

Reply:
Just checking the file, appears to be fixed.